arrocke/gloss-translation
# feat: track gloss history

## What has changed


- Created the `GlossHistory` table as described in issue #340 
- Whenever a user updates a gloss, a row is added to this table
- Whenever the user imports glosses from the old site, a ~~column (🤣)~~ row  is created for each gloss imported.
## Connected Issues
closes #340


## QA Steps

1. Reset the database to use the updated schema (`nx run db:prisma migrate reset`)
2. Query the contents of the `GlossHistory` table. Confirm that it is empty.
3. Change the gloss text for a word. Confirm that a new column is added to the `GlossHistory` table with the updated gloss text.
4. Then, change the gloss approval state without changing the gloss text. Confirm that a new column in `GlossHistory` is created with the updated approval state and `null` for the gloss text.

## Post-Deployment
